cumulus/polkadot-parachain/polkadot-parachain-lib/src/fake_runtime_api/mod.rs

```diff
@@ -17,5 +17,50 @@
 //! In an ideal world this would be one runtime which would simplify the code massively.
 //! This is not an ideal world - Polkadot Asset Hub has a different key type.
 
-pub mod asset_hub_polkadot_aura;
-pub mod aura;
+mod utils;
+
+use crate::common::types::CustomBlock;
+use utils::{impl_node_runtime_apis, imports::*};
+
+pub mod asset_hub_polkadot {
```

Comment:
In general would expect us to be more diligent in documenting this `lib` crate, as it is a public API that is meant to be used by externals. 

Perhaps try adding `#![deny(undocumented_pub_items)]` for this crate, and let it double check your choices?

Reply:
I would also hope the main `lib.rs` docs that go into `docs.rs` to be very extensive, and contain full examples of how the lib can be used. 

You can either do this here, or make an issue for it and assign me, I like doing such parts of the work :)

Reply:
The `fake_runtime` is not public. The `mod` is private to the crate. But, good point, I added `#![deny(missing_docs)]` and there were some errors.

Added this issue for documentation: https://github.com/paritytech/polkadot-sdk/issues/5464 . I would prefer it to do it in a separate PR, since this one is already pretty verbose.